Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Vrykul Ethnicity

## Changelog:
- Created a custom ethnicity for Vrykul and their subspecies (Kvaldir, Frost Vrykul, Drust).


# How to test:
There are two ways you can test this:
1. Open up Ruler Designer and generate male and female Vrykul, Frost Vrykul, Kvaldir, and Drust to see what they look like. 
2. Alternatively, load up into a game and look for these cultures in-game by using the character search panel.

Give feedback and suggestions on what, if anything, could or should be improved or changed.
